### ISSUE TITLE
Life events error: HTTP 500 with PostgreSQL backend (Monica 2.8.0)

### ISSUE THREAD

**Issue by LorenzoAncora** [monicahq/monica#1867]
With the PostgreSQL backend the creation of a new event is blocked by the **HTTP 500 error**:

```
SQLSTATE[42703]: Undefined column: 7 ERROR: column "profile_active_tab" of relation "users" does not exist LINE 1: update "users" set "updated_at" = $1, "profile_active_tab" =... ^ (SQL: update "users" set "updated_at" = 2018-10-01 21:32:21, "profile_active_tab" = life-events, "profile_new_life_event_badge_seen" = 1 where "id" = 1)
```

The migration (`php artisan migrate`) from the previous stable version is in turn blocked by the error:

``` 
 SQLSTATE[23502]: Not null violation: 7 ERROR:  null value in column "specific_information_structure" violates not-null constraint
  DETAIL:  Failing row contains (8, 2, new_relationship, null, f, 2018-10-01 21:27:49, 2018-10-01 21:27:49). (SQL: insert into "default_life_event
  _types" ("translation_key", "default_life_event_category_id", "created_at", "updated_at") values (new_relationship, 2, 2018-10-01 21:27:49, 2018
  -10-01 21:27:49))
```

This seems caused by new feature #1762.

**Comment by sylvainmetayer** [comment]
I'm having the same issue with a MySQL backend

**Comment by LorenzoAncora** [comment]
@djaiss it this an unsolvable problem? :-)

**Comment by LorenzoAncora** [comment]
Also found in #1893.

**Comment by LorenzoAncora** [comment]
@sylvainmetayer version 2.8.1 solved this problem for me. Do you still have the problem?

**Comment by sylvainmetayer** [comment]
@LorenzoAncora problem is also solved for me with 2.8.1

**Comment by LorenzoAncora** [comment]
> @sylvainmetayer version 2.8.1 solved this problem for me. Do you still have the problem?
>> @LorenzoAncora problem is also solved for me with 2.8.1

Good, then problem is fixed. We can close this issue. TY to all.
